_posts/2023-03-22-#27050.md

```diff
@@ -10,15 +10,59 @@ status: upcoming
 commit:
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
-## Questions
-1. Did you review the PR? [Concept ACK, approach ACK, tested ACK, or NACK](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#peer-review)? What was your review approach?
--->
+- [BIP 141
+  (SegWit)](https://github.com/bitcoin/bips/blob/master/bip-0141.mediawiki)
+  introduced a new structure called a `witness` that is committed to blocks
+  separately from the transaction merkle tree. Witnesses contain data required
+  to check transaction validity but not required to determine transaction
+  effects (output consumption/creation). In other words, witnesses are used to
+  validate the blockchain state not to determine what that state is.
+
+  - Witnesses are commited to by placing the root of the witness merkle tree in
+    the block's coinbase transaction. By doing so, the witnesses are commited to
+    in the transaction merkle tree through the coinbase transaction. Nesting the
+    witness commitment in the coinbase transaction was done to make SegWit
+    soft-fork compatible.
+
+- `Assume-valid` is a node setting that makes the node skip transaction
+  validity checks (signature and script checks) prior to a pre-determined
+  "known to be good" block (assume-valid point). The default assume-valid point
```

Comment:
Maybe specify "known to be good" to be "has a lot of work"

Reply:
I don't think there is any work requirement on the assume-valid point itself, e.g. we could set it to a low-work point and that would still work (e.g. if we never update the value in a release).